docs/tools_cheatsheet/Makefile.md

```diff
@@ -0,0 +1,142 @@
+**A simple Makefile**
+
+```Makefile
+    CC  =	gcc
+
+    NAME  =	binary_name
+
+    CFLAGS  =	 -Wpedantic -pedantic-errors -w  -Wextra  -Wall -Iinclude
+
+    SRC = core/my_printf.c \
+        core/save_printf.c \
+	    src/my_putstr.c	   \
+	    src/my_putchar.c   \
+    	src/my_strlen.c	   \
+        main.c
+
+    OBJ	=	$(SRC:.c=.o)
+
+    all:  $(NAME)
+
+    $(NAME):	$(OBJ)
+	    $(CC) -c $(OBJ) -o $(NAME)
+
+    clean:
+	    -rm -f $(OBJ)

```

Comment:
Remove the dash before `rm`

Reply:
It means to ignore errors, see [GNU Make](https://www.gnu.org/software/make/manual/html_node/Errors.html). 
Should I remove this anyway?

Reply:
I didn't know that 😅. I think that in basic makefile, you should see the errors. So, yes, just remove it.